library/core/src/mem/mod.rs

```diff
@@ -27,6 +27,10 @@ mod valid_align;
 // alignment as a parameter, such as `Layout::padding_needed_for`.
 pub(crate) use valid_align::ValidAlign;
 
+mod transmutability;
+#[unstable(feature = "transmutability", issue = "none")]
+pub use transmutability::{Assume, BikeshedIntrinsicFrom};
```

Comment:
Please also add tests checking that these are correctly feature gated

Reply:
Resolved by https://github.com/rust-lang/rust/pull/92268/commits/ebc233b04532d43554fce7e3bb0837c810d8935c and https://github.com/rust-lang/rust/pull/92268/commits/fe8f757e12f11943baccfd59b43a5166a9fe87ab